aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -211,6 +237,28 @@ module aptos_framework::aptos_governance {
         borrow_global<GovernanceConfig>(@aptos_framework).required_proposer_stake
     }
 
+    #[view]
+    /// Return remaining voting power of a stake pool on a proposal.
+    public fun get_remaining_voting_power(stake_pool: address, proposal_id: u64): u64 acquires VotingRecords, VotingRecordsV2 {
+        if (features::partial_governance_voting_enabled()) {
+            assert!(exists<VotingRecordsV2>(@aptos_framework), error::invalid_state(EPARTIAL_VOTING_NOT_INITIALIZED));
+        };
+        let record_key = RecordKey {
+            proposal_id,
+            stake_pool,
+        };
+        let voting_records = borrow_global<VotingRecords>(@aptos_framework);
```

Comment:
Let's just add a getter function for this as well has_entirely_voted or something like that that returns true if a stake pool has voted as tracked by VotingRecords (v1)